tests/integ/sagemaker/lineage/conftest.py

```diff
@@ -651,6 +659,8 @@ def static_endpoint_context(sagemaker_session, static_pipeline_execution_arn):
         )
         endpoint_arn = get_endpoint_arn_from_static_pipeline(sagemaker_session)
 
+    endpoint_arn = get_endpoint_arn_from_static_pipeline(sagemaker_session)
```

Comment:
It looks like we are overwriting endpoint_arn here

Reply:
removed